### PR TITLE
Add AssetIcon, DynamicIcon component & icon svgs

### PR DESCRIPTION
**Pull request by skyorion427**

### CODE REVIEW

File: src/assets/svg/telegram.svg [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
```
Comment (veado) [inline src/assets/svg/telegram.svg:1]: Don't override `telegram.svg` we already have in `master`! An `id` has been added to it manually, which is needed for theming.

File: src/components/uielements/assets/dynamicAsset/types.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+export type Size = 'small' | 'normal' | 'big'
+
+export type Sizes = {
+  [key: string]: string
```
Comment (veado) [inline src/components/uielements/assets/dynamicAsset/types.ts:4]: ` [key in Size]: string`

File: src/components/uielements/assets/dynamicAsset/types.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+}
+
+export type FontSizes = {
+  [key: string]: string
```
Comment (veado) [inline src/components/uielements/assets/dynamicAsset/types.ts:8]: `[key in Size]: string`

File: src/components/uielements/assets/assetIcon/assetIcon.style.ts [hunk 4]
```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+import { palette, key } from 'styled-theme'
+
+import { Props } from './assetIcon'
+
+const sizes: {
+  [key: string]: string
+} = {
```
Comment (veado) [inline src/components/uielements/assets/assetIcon/assetIcon.style.ts:8]:
```

import { Size } from './types'

type Sizes = {
  [key in Size]: string
}
const sizes: Sizes = ...

````

File: src/components/uielements/assets/assetIcon/assetIcon.style.ts [hunk 5]
```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components'
+import { palette, key } from 'styled-theme'
+
+import { Props } from './assetIcon'
```
Comment (veado) [inline src/components/uielements/assets/assetIcon/assetIcon.style.ts:4]: Don't re-import `Props` from `assetIcon` which will cause a circular dependency

```
assetIcon -> assetIcon.style -> assetIcon
```

Comment (skyorion427) [reply src/components/uielements/assets/assetIcon/assetIcon.style.ts:4]: Fixed

File: src/components/uielements/assets/assetIcon/assetIcon.style.ts (hunk 4 above)
Comment (skyorion427) [reply src/components/uielements/assets/assetIcon/assetIcon.style.ts:8]: Fixed

File: src/components/uielements/assets/dynamicAsset/types.ts (hunk 2 above)
Comment (skyorion427) [reply src/components/uielements/assets/dynamicAsset/types.ts:4]: Fixed

File: src/assets/svg/telegram.svg (hunk 1 above)
Comment (skyorion427) [reply src/assets/svg/telegram.svg:1]: Fixed

File: src/components/uielements/assets/assetIcon/assetIcon.style.ts [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+  [key in Size]: string
+}
+
+const sizes: Sizes = {
+  big: key('sizes.coin.big', '40px'),
+  small: key('sizes.coin.small', '30px')
+}
+
+export const AssetIconWrapper = styled.div`
+  width: ${({ size }: { size: Size }) => sizes[size || 'big']};
```
Comment (veado) [inline src/components/uielements/assets/assetIcon/assetIcon.style.ts:16]:
```
types Props = {
   size: Size
}

...


${(props: Props) => sizes[props.size || 'big']};
```

^ for all props ;)

File: src/components/uielements/assets/dynamicAsset/dynamicAsset.style.ts [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+  size?: Size
+  className?: string
+}
+
+type ExtraProps = {
+  startCol: string
+  stopCol: string
+}
+
+type Props = ComponentProps & ExtraProps
```
Comment (veado) [inline src/components/uielements/assets/dynamicAsset/dynamicAsset.style.ts:17]: Why do you combine `Props` from two different types and not just define everything in `Props`?

**Pull request merged**